app/src/community/google-services.json

```diff
@@ -1,162 +1,140 @@
 {
```

Comment:
Please remove these changes.

Reply:
@kbagchiGWC Everything looks good! Just remove those google-service file changes. Oh and please rebase with master as I just merged another PR. Thanks!